ui/ui_test.go

```diff
@@ -0,0 +1,33 @@
+package pitchforkui
+
+func ExamplePfUI_Page_show() {
```

Comment:
We'll need to update the call in PfUI from Page_show to PageShow for this to work; which will have global reach throughout all UI code as it is called from everywhere.

We can file a ticket for this, because if we do that that in this commit it will be even dirtier.